type_lens/type_view.py

```diff
@@ -161,7 +161,7 @@ def is_required(self) -> bool:
     @property
     def is_tuple(self) -> bool:
         """Whether the annotation is a ``tuple`` or not."""
-        return self.is_subtype_of(tuple)
+        return self.is_subclass_of(tuple)
```

Comment:
per the test `TypeView(...).is_tuple` would fail. this function doesn't need generics handling, so that seems fine to me?